analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -781,9 +782,14 @@ def signal_handler(signum, _):
                    'reproducer': reproducer_dir,
                    'ctu_connections': ctu_connections_dir}
 
+    @lru_cache
+    def __analyzer_config_map_get(analyzer_type):
+        return analyzer_config_map.get(analyzer_type)
+
+
     analyzed_actions = [(actions_map,
                          build_action,
-                         analyzer_config_map.get(build_action.analyzer_type),
```

Comment:
`analyzer_config_map` is not a regular and cheap `Dict`, it is a `multiprocess.managers.DictProxy` syncing its items through multiprocess environment. So just cache it.